src/annotator/sidebar.tsx

```diff
@@ -174,6 +174,7 @@ export class Sidebar implements Destroyable {
       this.iframeContainer = document.createElement('div');
       this.iframeContainer.style.display = 'none';
       this.iframeContainer.className = 'sidebar-container';
+      this.iframeContainer.id = 'sidebar-container';
```

Comment:
The iframe container is also the element containing the toggle button. According to the spec, it is ok that the toggle controls a parent element.

I decided to "link" it with this element because that's the element which styles get affected by the toggle, but I also considered referencing the iframe only. I think both options would be semantically correct.

Reply:
> According to the spec, it is ok that the toggle controls a parent element.

Do you have a reference for this? Functionally this seems to work correctly in VoiceOver, though I did wonder if it could cause confusion that a toggle button is part of "hidden" content but is not actually hidden.

Reply:
Well, maybe I wrote that in a way that it sounds like it is an absolute certainty, but it's actually my interpretation of https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

There it says that `aria-controls` has to reference the element "whose contents or presence are controlled by the element on which this attribute is set". Since this button actually handles a parent element, I came to the conclusion that it should reference that element.

Reply:
OK, that's just a general description of `aria-controls` which is applies to various uses. I wouldn't interpret it as a claim either way that constraints on the relationship between controlling / controlled elements. The spec at https://w3c.github.io/aria/#aria-controls doesn't identify any constraints either. In short, what we're doing here isn't explicitly forbidden, so I think we're OK.